ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/loopbacks-interfaces.md

```diff
@@ -82,6 +82,7 @@ interface Loopback0
 interface Loopback1
    description VTEP_VXLAN_Tunnel_Source
    ip address 192.168.254.3/32
+   isis bfd
```

Comment:
As mentioned in a previous comment, we should not enable isis bfd under a loopback interface.

Reply:
moved to loopback99